src/ddscxx/src/org/eclipse/cyclonedds/pub/qos/DataWriterQosDelegate.cpp

```diff
@@ -144,6 +144,13 @@ DataWriterQosDelegate::policy(const dds::core::policy::WriterDataLifecycle& life
     lifecycle_ = lifecycle;
 }
 
+void
```

Comment:
Please also encapsulate this whole body with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro.

Reply:
Will do.